### ISSUE TITLE
Publish binaries of releases 

### ISSUE THREAD

**Issue by plevold** [lfortran/lfortran#965]
It would be nice if you could publish binaries for Linux and Windows of the LFortran releases.

I've just started experimenting with LFortran. Enjoying so far, but I'm not really interested in the Jupyter integration. Being able to download just the binaries without depending on Conda would significantly lower the barrier to entry.

**Comment by certik** [comment]
Yes, we did this on GitLab, but don't do it on GitHub. If you or anyone else wanted to contribute it, that would be awesome.

**Comment by gnikit** [comment]
Let me know if you want help with the CI stuff. I can draft up a GA workflow pretty quick, feel free to assign this to me @certik and I'll have a look.

**Comment by certik** [comment]
@gnikit thank you, that would be awesome! I just gave you push access, so you should have all the permissions to set this up.

**Comment by gnikit** [comment]
Thanks, will try and get a draft later tonight.

**Comment by gnikit** [comment]
@certik So I did this a while back but there is a runtime failure when building the binaries which I suppose has to do with the GitHub Action runners

PR https://github.com/gnikit/lfortran/pull/2
Failure in CI: https://github.com/gnikit/lfortran/actions/runs/3388505318/jobs/5630598077#step:8:190
Failure in CI (when including libxeus, which probably we shouldn't): https://github.com/gnikit/lfortran/actions/runs/3388663907/jobs/5630934570#step:8:188

**Comment by certik** [comment]
Thanks! Yes, let's build without XEUS at first. I am not sure what the other issue is, we'll have to debug it.
